mixer/adapter/svcctrl/config/config.proto

```diff
@@ -17,13 +17,38 @@ syntax = "proto3";
 package adapter.svcctrl.config;
 
 import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
 
 option go_package = "config";
 option (gogoproto.goproto_getters_all) = false;
 option (gogoproto.equal_all) = false;
 option (gogoproto.gostring_all) = false;
 
-message Params {
+// Adapter runtime config paramters.
+message RuntimeConfig {
+    int32 check_cache_size = 1;
+    google.protobuf.Duration check_result_expiration = 2;
+}
+
+message QuotaConfig {
+    string quota_name = 1;
+    string google_quota_metric_name = 2;
+    google.protobuf.Duration expiration = 3;
+}
+
+// Config for a managed GCP service.
+message ServiceConfig {
```

Comment:
ServiceConfig is overloaded. Please use a specific name such as GcpServiceSetting.